include/lsst/shapelet/MultiShapeletFunction.h

```diff
@@ -71,6 +71,8 @@ class MultiShapeletFunction {
 
     MultiShapeletFunction() : _components() {}
 
+    MultiShapeletFunction(MultiShapeletFunction const & other) = default;
```

Comment:
Also add move and move assignment (and then also copy assignment) constructors unless there is a compelling reason not to.